ote_cli/ote_cli/tools/train.py

```diff
@@ -133,8 +147,51 @@ def main():
             },
         )
 
+    task_module = task_class.__module__.split(".")[0]
+    lr_finder_enabled = None
+    if args.enable_hpo and check_hpopt_available():
```

Comment:
I would not like to see this logic in `train.py`, hide it somewhere in utils at least, or invent something better

Reply:
I made a method "run_hpo" and moved all HPO-related logic into the method. Thank you!